src/components/BulletChart.tsx

```diff
@@ -0,0 +1,65 @@
+// (C) 2007-2020 GoodData Corporation
+import * as React from "react";
+import omit = require("lodash/omit");
+import { VisualizationInput, VisualizationObject } from "@gooddata/typings";
+
+import { Subtract } from "../typings/subtract";
+import { BulletChart as AfmBulletChart } from "./afm/BulletChart";
+import { ICommonChartProps } from "./core/base/BaseChart";
+import { convertBucketsToAFM } from "../helpers/conversion";
+import { getStackingResultSpec } from "../helpers/resultSpec";
+import { MEASURES, ATTRIBUTE } from "../constants/bucketNames";
+import {
+    getViewByTwoAttributes,
+    sanitizeConfig,
+    disableBucketItemComputeRatio,
+} from "../helpers/optionalStacking/common";
+
+export interface IBulletChartBucketProps {
+    primaryMeasure: VisualizationInput.AttributeOrMeasure;
+    viewBy?: VisualizationInput.IAttribute | VisualizationInput.IAttribute[];
```

Comment:
'VisualizationInput.IAttribute' in BarChart is just for backward compatibility when multiple items for ViewBy were introduced.
Here in BulletChart you can use just 'VisualizationInput.IAttribute[]' if you want to simplify it

Reply:
OK, good to know. So we'll make it support just an array.